src/components/Content/Content.js

```diff
@@ -1,18 +1,112 @@
 import './Content.css';
-import Counter from './Counter/Counter';
+import Ingredient from './Ingredient/Ingredient';
+import Builder from './Builder/Builder';
 import React from "react";
 
-function Content() {
+class Content extends React.Component{
+
+    constructor (props) {
+        super(props);
+
+        this.state = {
+            ingredients: this.toState(),
+            total: "1.00",
+            isModalActive: false
+        }
+
+        this.removeFromState = this.removeFromState.bind(this);
+        this.addToState = this.addToState.bind(this);
```

Comment:
What about binding `toState` and `bill`? They are called methods from `this`, so they will work. But if they want to be reusable? Or call from `onClick` - it will be hard to maintain in that case